docs/systemd.md

```diff
@@ -24,12 +24,20 @@ After=network.target
 # Requires=puma.socket
 
 [Service]
-# Foreground process (do not use --daemon in ExecStart or config.rb)
-Type=simple
+#Puma automatically supports systemd's `Type=notify` and watchdog service
```

Comment:
Automatically isn't entirely true. You should mention you also need the correct gem installed.

Reply:
Ok, but one question, since I've added the gem in Puma's Gemfile, will it not be installed?

Reply:
No, the Gemfile is not the same thing as the gemspec. `sd_notify` will be an optional dependency as it will only be needed in certain environments (e.g. if you run on Heroku, you don't need it and don't want to spend time installing it on each deploy)

Reply:
Thanks @dentarg for the clarification!